addons/project/models/project.py

```diff
@@ -950,6 +951,10 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     # Tracking of this field is done in the write function
     user_ids = fields.Many2many('res.users', relation='project_task_user_rel', column1='task_id', column2='user_id',
         string='Assignees', default=lambda self: self.env.user)
+    # Display the assignees of the task in project sharing feature
+    portal_user_ids = fields.Many2many('res.users', relation='project_task_user_rel', column1='task_id', column2='user_id',
+        string='All Assignees', compute='_compute_portal_user_ids', compute_sudo=True,
+        readonly=False, search='_search_portal_user_ids', default=lambda self: self.env.user)
```

Comment:
Why should this field be editable?

Reply:
Because in project sharing feature, we allow the portal user to assign himself in a task for instance.

Reply:
Why not just handle that in the "Assign To Me" button's action?

Reply:
Indeed, we could use two buttons to allow the portal user to assign and unassign himself. But we could do the same with this field, I don't see the problem if I do it like this. Perhaps, because of the lines added in `create` and `write` methods ? 🤔
Moreover if another portal user is in the same company, then he could assign this one, for instance.

If you prefer to use two buttons to do this, then I will change this.

Reply:
>  But we could do the same with this field, I don't see the problem if I do it like this.

Reply:
Ok thanks for the explanation. I will do two buttons, one to assign the current user to the task and the other one to unassign the current user.